k8s/helm/values.yaml

```diff
@@ -10,6 +10,7 @@ image:
   pullPolicy: Always
 
 submissionBucket: ""
+stackdriverProject: ""
```

Comment:
You should add this to
https://github.com/ONSdigital/eq-survey-runner/blob/v3/k8s/deploy_app.sh
and update the README.